src/app/components/pages/wallets/address-detail/wallet-detail.component.scss

```diff
@@ -123,3 +127,66 @@
     }
   }
 }
+
+@keyframes floatup {
+  20% {
+    opacity: .999
+  }
+
+  100% {
+    -webkit-transform: translate3d(-50%, -17px, 0);
+    transform: translate3d(-50%, -17px, 0)
+  }
+}
+
+@-webkit-keyframes floatup {
+  20% {
+    opacity: .999
+  }
+
+  100% {
+    -webkit-transform: translate3d(-50%, -17px, 0);
```

Comment:
This line is not necessary, the Sass compiler is responsible for placing the prefix.